SampleProject/VOKCoreDataManagerTests/VOKThreadingTests.m

```diff
@@ -0,0 +1,37 @@
+//
+//  VOKThreadingTests.m
+//  Vokoder Sample Project
+//
+//  Created by Brock Boland on 2/26/16.
+//  Copyright © 2016 Vokal. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#import <VOKCoreDataManager.h>
+
+@interface VOKThreadingTests : XCTestCase
+
+@end
+
+@implementation VOKThreadingTests
+
+- (void)testSetupOnBackgroundThreadFails
+{
+    XCTestExpectation *willFinish = [self expectationWithDescription:@"Background thread setup will finish"];
+
+    // Setup on the main thread should work
+    [[VOKCoreDataManager sharedInstance] resetCoreData];
+    XCTAssertNoThrow([[VOKCoreDataManager sharedInstance] setResource:@"VOKCoreDataModel" database:nil]);
+
+    NSOperationQueue *backgroundQueue = [[NSOperationQueue alloc] init];
+    backgroundQueue.qualityOfService = NSOperationQualityOfServiceBackground;
+    [backgroundQueue addOperationWithBlock:^{
+        [[VOKCoreDataManager sharedInstance] resetCoreData];
+        // Setup on a background thread should fail
```

Comment:
Those have been around since iOS 5, so why was that warning added to the comments last May? Lingering superstition? Or did #72 and #76 modernize things enough in the interim such that said warning can be removed? If so, I'm assuming that the [other main queue assertion](https://github.com/vokal/Vokoder/blob/master/Pod/Classes/VOKCoreDataManager.m#L484) can be dropped as well.